### PR TITLE
WIP: CodeRefinery Zulip guide updates

### PR DESCRIPTION
**Pull request by samumantha**
Changes to be introduced in this PR:
- channel cleanup (merge and split, based on feedback)
- streams are now channels (official Zulip change)
- default channels update
- tips and tricks section expanded

TODO:
- [x] transform table to bullet points for readability
- [x] Fill todos in text
- [x] rename all stream mentions in this doc to channel
- [x] **Do the actual updates on Zulip first!**
    - [x] Rename channels
    - [x] Create new ones
    - [x] Archive/move others according to list in this PR
        - [x] #workshops to #events and #tools-workshop then remove workshops
        - [x] decide what happens to #announce and do it
        - [x] decide what happens to zulip and do it
- [x] Update channel descriptions (in Zulip and text)
- [x] Update default subscribed channels in Zulip and text
- [x] Add encouragement to add your own channel (eg regional!)
    - this is why #finland exists and not other by country channels 
    - it is in channel description
- [x] check and if necessary update organization description in Zulip

Anyone is welcome to add your tips and tricks on how to use Zulip here! Please help make this guide a helpful resource.

### CODE REVIEW

**Comment by samumantha** [comment]
Note: workshop-chat now archived:workshop chat, everyone got unsubscribed, admin only writing rights, but messages can still be found.

**Comment by samumantha** [comment]
TIW channel has been moved to general (multiple TIW topics as they were in TIW), but in future better to use common #general > TIW (today I watched) topic. Still considering to move old messages there too.

**Comment by samumantha** [comment]
Carpentries channel was archived (only old info in it) other still current topics have been moved to CodeRefinery and lessons channel

**Comment by samumantha** [comment]
Monday morning hello moved from #general to #greetings

**Comment by samumantha** [comment]
A new suggestion for a channel: #funding for sharing funding opportunities and looking for collaborators etc; what do you think?

**Comment by samumantha** [comment]
And should we keep #announce for Zulip changes, new channels and other announcements that concern all in chat (like the chat cleanup announcement)?
So that only admins can post?

**Comment by samumantha** [comment]
#new members content is all in #greetings

**Comment by samumantha** [comment]
kickstart-aalto is not hpc-kickstart

File: chat.md [hunk 1]
```diff
@@ -31,61 +29,94 @@ on that chat, then selecting what you are interested in.**
...
+| aalto                         | Aalto University scientific computing community (+friends)                                                                                          | archive                      |
+| aaltoscicomp                  | Aalto scientific computing support personnel                                                                                                        | archive                           |
+| advent-of-code                | For chatting about the advent of code challenges                                                                                                    | - (keep as is)                          |
+| cr-retreat                    | Temporary private stream for CodeRefinery writing retreat related discussions                                                                       | (keep as is, archive next week)         |
+| finland                       | Discussion related to activities in Finland (CodeRefinery, RSE, HPC, SciComp, anything else)                                                        | **remove**, or add other countries too? |
```
Comment (samumantha) [inline chat.md:63]:
```suggestion
| finland                       | Discussion related to activities in Finland (CodeRefinery, RSE, HPC, SciComp, anything else)                                                        | - |
```

**Comment by samumantha** [comment]
#tools-workshop cleaned, all lesson realted discussion is in #lesson channel. 
Everything old is marked resolved and similar topics are merged

**Comment by samumantha** [comment]
private #tools-workshop-chat archived, was all outdated information, workshop specific chats happen in other channels

**Comment by samumantha** [comment]
topics of #TIL moved to #general with TIL in beginning of topic header

**Comment by samumantha** [comment]
#infrastructure contents moved to coderefinery and help channels

File: chat.md [hunk 2]
```diff
@@ -31,61 +29,94 @@ on that chat, then selecting what you are interested in.**
...
+| SWC DLN/ELIXIR/UiB Dlab       | ?                                                                                                                                                  | (keep as is)                            |
+| coderefinery-tools-workshop                | Where we organize the "standard" CodeRefinery workshop each spring and autumn.                                                                      | keep as is       |
+| workflows-course              | Real-life cluster computing workflows course, "Tuesday tools and techniques for HPC". A new course in 2024                                          | keep as is                |
+| hpc | Discussions around HPC in Nordics and beyond | -|
+| zulip-admin | Zulip default notifications | - |
+
+
+
 
 You can make topical private streams for groups that significantly
```
Comment (samumantha) [inline chat.md:80]:
```suggestion
We also encourage you to create your own channels for sub-communities, e.g. by region (such as #finland). You can make topical private streams for groups that significantly
```

**Comment by samumantha** [comment]
#workshops went to #tools-workshop where discussion was about different iterations about the general codeRefinery workshop, other workshop discussions went to events, and some other non workshop related discussions went into coderefinery or lessons channels

**Comment by samumantha** [comment]
#announce moved to where it fit, #events, #jobs, #general, new channel notification set to appear in #general now; #announce archived

**Comment by samumantha** [comment]
#zulip set to have Zulip announcements, currently not a default channel

**Comment by bast** [comment]
> Please add your tips and tricks !

I have it on my high-prio list to review and to add how I work with Zulip but it might take few days. Thanks a lot for all the work!

**Review by bast — APPROVED** [review]

**Comment by bast** [comment]
Looks like I broke spelling. Fixing ...

**Comment by samumantha** [comment]
Thanks, merging for now; but if you have more please add :)

**Pull request merged**